src/query_engine/query_engine.cpp

```diff
@@ -86,18 +102,20 @@ std::unique_ptr<BoolExpression> parse_expression(const Database& db, const rapid
       }
       if (exact >= 0) {
          return std::make_unique<NucEqEx>(position, value);
-      } else { // Approximate query!
+      } else {  // Approximate query!
          return std::make_unique<NucMbEx>(position, value);
       }
    } else if (type == "NucMut") {
       assert(js.HasMember("position"));
       unsigned pos = js["position"].GetUint();
       char ref_symbol = db.global_reference[0].at(pos);
       /// this <= is correct! the negation would flip the exact bit from -1 to +1 and vice versa
-      if (exact <= 0) { /// NucEqEx
-         return std::make_unique<NegEx>(std::make_unique<NucEqEx>(pos, silo::to_symbol(ref_symbol)));
-      } else { /// NucMbEx
-         return std::make_unique<NegEx>(std::make_unique<NucMbEx>(pos, silo::to_symbol(ref_symbol)));
+      if (exact <= 0) {  /// NucEqEx
+         return std::make_unique<NegEx>(std::make_unique<NucEqEx>(pos, silo::to_symbol(ref_symbol))
+         );
```

Comment:
This looks odd. Can we somehow make it look like this?
```suggestion
         return std::make_unique<NegEx>(
            std::make_unique<NucEqEx>(pos, silo::to_symbol(ref_symbol))
         );
```

Reply:
Good idea. However, with the clang-tidy pullrequest, this should not occur at this point due to renaming.